src/Media.Plugin/UWP/NewMediaImplementation.cs

```diff
@@ -440,8 +443,6 @@ Task<bool> ResizeAsync(StorageFile file, StoreCameraMediaOptions mediaOptions)
                 if (photoSize == PhotoSize.Full)
                     return Task.FromResult(false);
 
-				var customPhotoSize = mediaOptions.CustomPhotoSize;
-				var quality = mediaOptions.CompressionQuality;
```

Comment:
It was never used.

Reply:
And customPhotoSize was only used once too, in line 462.

Reply:
the quality?

Reply:
The quality variable, mediaOptions.CompressionQuality is used instead.